internal/cmd/iam/command_groupmapping_update.go

```diff
@@ -0,0 +1,72 @@
+package iam
+
+import (
+	"github.com/spf13/cobra"
+
+	ssov2 "github.com/confluentinc/ccloud-sdk-go-v2/sso/v2"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+)
+
+func (c *groupMappingCommand) newUpdateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "update <id>",
+		Short:             "Update a group mapping.",
+		Args:              cobra.ExactArgs(1),
+		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validArgs),
+		RunE:              c.update,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Update the description of group mapping "pool-123456".`,
+				Code: `confluent iam group-mapping update pool-123456 --description "Update demo group mapping information."`,
+			},
+		),
+	}
+
+	cmd.Flags().String("name", "", "Name of the group mapping.")
+	cmd.Flags().String("description", "", "Description of the group mapping.")
+	cmd.Flags().String("filter", "", "A supported Common Expression Language filter expression for group mappings.")
```

Comment:
```suggestion
	cmd.Flags().String("filter", "", "A supported Common Expression Language (CEL) filter expression for group mappings.")
```

Reply:
sounds good! looks like `iam pool` uses this too